Week-3/4_Voting/OneVoteRound.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+contract oneVotingRound {
+    address admin;
+    bool votingInProgress;
+    bool votingStopped;
+    uint128 totalRegisteredCandidates;
+
+    address leadingCandidate;
+    uint256 leadingCandidateVotes;
+
+    struct Candidate {
+        address addr;
+        uint256 votes;
+    }
+
+
+    mapping(address => Candidate) CandidatesRecord; // to check if candidate is registered
+    mapping(address => bool) votersList; // to check if voters have voted or not
```

Comment:
`votersList` name does not convey what it stores. Consider naming it to: hasVoted or something else.

Reply:
Done